### PR TITLE
SW-2565 Remove Old Theme

### PR DESCRIPTION
**Pull request by charlie-tf**
- Remove directories and scripts for old theme
- Rename new theme to be the only one
- Upgraded typescript to work with terraware-web

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "eject": "react-scripts eject",
     "storybook": "start-storybook",
     "build-storybook": "build-storybook -s public",
-    "build-dictionary": "./style-dictionary/generate.sh && ./style-dictionary2/generate.sh",
+    "build-dictionary": "./style-dictionary/generate.sh && style-dictionary/generate.sh",
```
Comment (karthikbtf) [inline package.json:45]: we could remove the first command,, both do the same thing here

File: package.json [hunk 2]
```diff
@@ -107,7 +107,7 @@
     "tslint-eslint-rules": "^5.4.0",
     "tslint-react": "^5.0.0",
     "tslint-react-hooks": "^2.2.2",
-    "typescript": "^4.3.2"
+    "typescript": "^4.9.4"
```
Comment (karthikbtf) [inline package.json:110]: Just wanted call out the build warnings:

```
WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.5.0

YOUR TYPESCRIPT VERSION: 4.9.4

Please only submit bug reports when using the officially supported version.
```

File: src/style-dictionary-dist/TerrawareTheme.ts [hunk 3]
```diff
@@ -2,541 +2,535 @@ import { PaletteOptions } from '@mui/material/styles/createPalette';
...
+    TwClrTxtWarning?: React.CSSProperties['color'];
   }
 }
 
 type ThemeDictionary = {
   [key: string]: any;
 };
 
-const TerrawareTheme: ThemeDictionary = {
+const TerrawareTheme2: ThemeDictionary = {
```
Comment (karthikbtf) [inline src/style-dictionary-dist/TerrawareTheme.ts:359]: call this `TerrawareTheme`?

File: src/style-dictionary-dist/TerrawareTheme.ts [hunk 4]
```diff
@@ -2,541 +2,535 @@ import { PaletteOptions } from '@mui/material/styles/createPalette';
...
+    "TwClrIcnOnWarning": "#FFFFFF",
+    "TwClrIcnWarning": "#A36A2D",
+    "TwClrTxtOnWarning": "#FFFFFF",
+    "TwClrTxtWarning": "#A36A2D"
   }
 }
 
-export default TerrawareTheme;
\ No newline at end of file
+export default TerrawareTheme2;
```
Comment (karthikbtf) [inline src/style-dictionary-dist/TerrawareTheme.ts:536]: `TerrawareTheme`?

File: style-dictionary/build-theme.mjs [hunk 5]
```diff
@@ -31,9 +34,9 @@ const TERRAWARE_PALETTE = `type ThemeDictionary = {
   [key: string]: any;
 };
 
-const TerrawareTheme: ThemeDictionary = <TERRAWARE_PALETTE>
+const TerrawareTheme2: ThemeDictionary = <TERRAWARE_PALETTE>
```
Comment (karthikbtf) [inline style-dictionary/build-theme.mjs:37]: `TerrawareTheme`?

**Review by karthikbtf — APPROVED** [review]
Woot!! 🎉 Thanks so much!
I have a few comments in there.. feel free to merge when ready.

File: package.json (hunk 2 above)
Comment (karthikbtf) [reply package.json:110]: btw this was in terraware-web

File: package.json (hunk 1 above)
Comment (karthikbtf) [reply package.json:45]: basically
```
    "build-dictionary": "./style-dictionary/generate.sh",
```

Comment (charlie-tf) [reply package.json:45]: 🤦 yup!

**Pull request merged**